lms/static/js/search/base/views/search_results_view.js

```diff
@@ -14,7 +14,7 @@ define([
         // these should be defined by subclasses
         el: '',
         contentElement: '',
-        resultsTemplateId: '',
+        revisionListTemplateId: '',
```

Comment:
@bdero I think you did a mass rename? The changes in this file are false positives, please undo them :)

Same goes for the changes in `lms/static/js/search/course/views/search_results_view.js` and `lms/static/js/search/dashboard/views/search_results_view.js` below.